jni_gen/test/jackson_core_test/jnigen.yaml

```diff
@@ -0,0 +1,36 @@
+maven_downloads:
```

Comment:
Please add a section to the README.md with all the yaml config options. (Feel free to copy-paste the setup from package:ffigen's README.md)

Reply:
I thought I'd do that in next PR or so along with runnable examples.